public/app/features/dashboard/components/PanelEditor/PanelEditor.tsx

```diff
@@ -383,7 +300,13 @@ export class PanelEditorUnconnected extends PureComponent<Props> {
       <div className={styles.wrapper} aria-label={selectors.components.PanelEditor.General.content}>
         {this.editorToolbar(styles)}
         <div className={styles.verticalSplitPanesWrapper}>
-          {uiState.isPanelOptionsVisible ? this.renderWithOptionsPane(styles) : this.renderHorizontalSplit(styles)}
+          <ThreePaneSplit
+            horizontalSplitComponents={this.renderPanelAndEditor(styles)}
+            rightPaneComponent={this.renderOptionsPane()}
+            uiState={uiState}
+            updateUiState={updatePanelEditorUIState}
+            rightPanelVisible={uiState.isPanelOptionsVisible}
```

Comment:
```suggestion
            rightPaneVisible={uiState.isPanelOptionsVisible}
```

property should be rightPaneVisible